modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc

```diff
@@ -0,0 +1,276 @@
++
+The following example creates a stable service called `argo-rollouts-stable-service`:
++
+[source,yaml]
+----
+# Stable traffic is directed to this Service, for example, it might initially be 70% of traffic.
+apiVersion: v1
+kind: Service
+metadata:
+  name: argo-rollouts-stable-service #<1>
+spec:
+  ports:
+  - port: 80
+    targetPort: http
+    protocol: TCP
+    name: http
+
+  selector: # this field/sub-field must have the same value as Rollout 
+    app: rollouts-demo 
+    # Rollouts will update the Service with pod-template-hash of the stable (non-canary) ReplicaSet. e.g.:
+    # rollouts-pod-template-hash: 1b6a7733
+----
+<1> Name of the stable service created in the Services page
+.. Click *Create* to create the stable service.
++
+[IMPORTANT]
+====
+Ensure that the name of the stable service specified in the `Route` object matches with the name of the stable service specified in the `Service` object.
+====
+. Create the `Rollout` CR to reference the `Route` and `Service` objects.
+.. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators* -> *Red Hat OpenShift GitOps* -> *Rollout*.
+.. On the *Create Rollout* page, click *YAML view* and add the following snippet:
++
+The following example creates a `Rollout` CR called `rollouts-demo`:
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: rollouts-demo
+spec:
+  template:
+    metadata:
+      labels:
+        app: rollouts-demo
+    spec:
+      containers:
+      - name: rollouts-demo
+        image: argoproj/rollouts-demo:blue
+        ports:
+        - name: http
+          containerPort: 8080
+          protocol: TCP
+        resources:
+          requests:
+            memory: 32Mi
+            cpu: 5m
+                
+  revisionHistoryLimit: 2
+  replicas: 5
+  strategy:
+    canary:
+      canaryService: argo-rollouts-canary-service # must match name of Service above #<1>
+      stableService: argo-rollouts-stable-service # must match name of Service above #<2>
+      trafficRouting:
+        plugins:
+          argoproj-labs/openshift:          
+            routes: #<3>
+              - rollouts-demo-route  # must match name of Route, above
+      steps: # In this example, route 30%/60%/100% of traffic to canary
+      - setWeight: 30
+      - pause: {}
+      - setWeight: 60
+      - pause: {}
+  selector: # Contents of this field must match the same value in the canary and stable Services (same contents for both)
+    matchLabels:
+      app: rollouts-demo
+----
+<1> Rollout references the canary service
+<2> Rollout references the stable service
+<3> Rollout references the route
+.. Click *Create*. 
+.. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the *RolloutManager* instance shows as *Phase: Available*.
+
+. Verify that the route is directing 100% of the traffic towards the stable version of the application. 
```

Comment:
```suggestion
. Verify that the route is directing 100% of the traffic towards the stable version of the application. 
+
[NOTE]
====
Info about rollout from introduction.
====
```
Here is where I would put that info about `Rollout` resource routing all of the traffic towards the stable version of the application at first.

Reply:
Fixed.